app/forms/state_file/md_two_income_subtractions_form.rb

```diff
@@ -0,0 +1,24 @@
+module StateFile
+  class MdTwoIncomeSubtractionsForm < QuestionsForm
+    set_attributes_for :intake, :primary_student_loan_interest_ded_amount, :spouse_student_loan_interest_ded_amount
+
+    validate :valid_amounts
+
+    def save
+      @intake.update(attributes_for(:intake))
+    end
+
+    private
+
+    def valid_amounts
+      if @intake.direct_file_data.fed_student_loan_interest.present?
+        if (primary_student_loan_interest_ded_amount.to_f + spouse_student_loan_interest_ded_amount.to_f) != @intake.direct_file_data.fed_student_loan_interest.to_f
```

Comment:
say `fed_student_loan_interest` is 100 and the person puts in 50 and 49.5 (for some reason), do we want to be strict and have them correct that data to be 50? or should we round these inputs?